storybook/storybook-react/src/HeroImage/HeroImage.stories.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Grid } from '@amsterdam/design-system-react'
+import { SearchField } from '@amsterdam/design-system-react'
+import { HeroImage } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Media/Hero Image',
+  component: HeroImage,
+  args: {
+    children: <SearchField />,
+  },
+} satisfies Meta<typeof HeroImage>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    src: 'https://picsum.photos/1280/360?random=1',
+  },
+}
+
+export const WithSearchField: Story = {
+  args: {
+    src: 'https://picsum.photos/1280/360?random=2',
+    children: (
+      <Grid paddingVertical="medium">
```

Comment:
This is not needed if you use the grid-area trick, and you'll avoid misalignment with the grid that it could already be in.